packages/syft/src/syft/ast/klass.py

```diff
@@ -161,6 +164,86 @@ def run_class_method(
 
         return result
 
+    def run_class_smpc_method(
+        __self: Any,
+        *args: Tuple[Any, ...],
+        **kwargs: Any,
+    ) -> object:
+        """Run remote class method on a SharePointer and get pointer to returned object.
+
+        Args:
+            *args: Args list of class method.
+            **kwargs: Keyword args of class method.
+
+        Returns:
+            Pointer to object returned by class method.
+        """
+        # stdlib
+        from uuid import UUID
+
+        # third party
+        import numpy as np
+
+        # syft absolute
+        from syft.core.node.common.action.smpc_action_message import (
+            MAP_FUNC_TO_NR_GENERATOR_INVOKES,
+        )
+
+        # TODO: Seed should always be regenerated for every operation and sent
+        seed = 42
```

Comment:
How big of a project is it to handle this TODO before we merge. If we don't handle it let's definitely create a Story in Clubhouse before merging so we don't forget.

Reply:
It should not be that big.

The party that has the MPCTEnsor would send in the kwargs the seed and each party would generate the UUIDs

Reply:
Is this resolved?